flasksrc/schema.sql

```diff
@@ -48,3 +51,31 @@ CREATE TABLE toynet_quiz_options (
   FOREIGN KEY (quiz_id, question_id) REFERENCES toynet_quizzes (quiz_id, question_id)
 );
 
+
+-- surveys submodule
+
+CREATE TABLE toynet_surveys (
+  survey_id INTEGER NOT NULL,
+  question_id  INTEGER NOT NULL,
+  type TEXT NOT NULL,
+  question TEXT NOT NULL,
+  created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (survey_id, question_id)
+);
+
+CREATE TABLE toynet_survey_options (
+  survey_id INTEGER NOT NULL,
+  question_id INTEGER NOT NULL,
+  option_id INTEGER NOT NULL,
+  option TEXT NOT NULL,
+  PRIMARY KEY (survey_id, question_id, option_id)
+  FOREIGN KEY (survey_id, question_id) REFERENCES toynet_surveys (survey_id, question_id)
+);
+
+CREATE TABLE toynet_survey_units (
+  survey_id INTEGER NOT NULL,
+  question_id INTEGER NOT NULL,
+  unit TEXT NOT NULL,
+  PRIMARY KEY (survey_id, question_id, unit)
+  FOREIGN KEY (survey_id, question_id) REFERENCES toynet_surveys (survey_id, question_id)
+);
```

Comment:
Let's work on a different schema for this. We want to decouple `toynet_surveys`, `toynet_survey_questions`, and `toynet_survey_question_options` as our tables. And we can make `unit` a NULL-able field in `toynet_survey_questions`.

You'll likely hit an n:n table relationship issue between between the `toynet_survey_questions` and `toynet_survey_question_options` tables, so to avoid the additional complexity of [database normalization](https://en.wikipedia.org/wiki/Database_normalization#Satisfying_3NF) we can just have duplicate entries for text like "yes" and "no" so that we maintain a 1:n relationship.

It would also be better to have an enum-like construct in lieu of `type` (SQLite does not have enums, so a bit of research will get you a good solution).

These changes will help us reuse questions in future developments of Toynet as well as keep the overall complexity of the schema low.

Good catch `DROP`ing `toynet_quiz_questions` as well!